components/next_steps_view/steps.ts

```diff
@@ -116,13 +116,18 @@ export const getSteps = createSelector(
     (state: GlobalState) => isFirstAdmin(state),
     (currentUser, firstAdmin) => {
         let roles = currentUser.roles;
+        const isGuestUser = isGuest(currentUser);
```

Comment:
Check out this part: https://github.com/mattermost/mattermost-webapp/blob/066a51470cf23935aa63ef2018c2282c184a5dfa/components/next_steps_view/steps.ts#L36

We already have support for which steps should show based on role. I think all you need to do to fix this, is to put `system_user` in the roles array for that step.

Reply:
@nickmisasi by placing only system_user in the roles array, it hides the third step for the admin and enables it only for users. If I placed both, system_admin and system_user in the roles array, it shows it for admin but hides it for normal users. Any ideas why is this happening?